charts/testkube-logs/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "testkube-logs.fullname" . }}-test-connection"
+  labels:
+    {{- include "testkube-logs.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
```

Comment:
many customers use their own image registries and all the images should be referenced as variables so that they can be easily replaced